minio/minio
# Add expiration to ListServiceAccounts function

## Description
This commit adds expiration date parameter to the ListServiceAccounts function.

## Motivation and Context
This PR is required for minio/mc#4576 to work. It is also linked with minio/madmin-go#195 and minio/console#2824, so we would need to merge this one after all the aforementioned ones.

## How to test this PR?
- On go.mod, do the following:
  - Replace the `madmin-go` module with the branch on minio/madmin-go#195
  - Replace the `mc` module with the branch on minio/mc#4576
  - Replace the `console` module with the branch on minio/console#2824
- Build and run the MinIO Server
- Build `mc` as specified in minio/mc#4576, run the `mc admin user svcacct list ALIAS TARGET_ACCOUNT` command and verify service accounts are listed with their expiration dates.

## Types of changes

## Checklist:
